core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-graph.ts

```diff
@@ -69,10 +66,47 @@ export class WorkflowGraph {
     if (!operator) {
       throw new Error(`operator with ID ${operatorID} doesn't exist`);
     }
+    if (this.isOperatorDisabled(operatorID)) {
+      this.enableOperator(operatorID);
+    }
```

Comment:
I see the intention of enabling an operator so that you can delete it correctly. 
But I wonder if there are better ways to do this, several reasons:
1. Enable an operator would trigger a lot of events, such as workflow persist, validation, schema propagation, etc. All those are unnecessary overheads.
2. Would the user "see" any effect?

Reply:
changed to just delete from the map without trigger the event